package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
@YuryStrozhevsky looks like npm is already headed in that direction. ;-) As of V3 issues a warning when peer dependency isn't installed without the need to explicitly declare installation. Wanna try it out? https://docs.npmjs.com/files/package.json#peerdependencies